core/systems/promotion/landing_page.ex

```diff
@@ -144,11 +144,13 @@ defmodule Systems.Promotion.LandingPage do
 
         <Text.title2 margin=""><%= dgettext("eyra-promotion", "expectations.public.label") %></Text.title2>
         <.spacing value="M" />
-        <Text.body_large><%= @vm.expectations %></Text.body_large>
+        <%!-- These fields use the raw() helper to make sure contents from the wysiwyg are rendered --%>
+        <Text.body_large><%= raw(@vm.expectations) %></Text.body_large>
         <.spacing value="M" />
         <Text.title2 margin=""><%= dgettext("eyra-promotion", "description.public.label") %></Text.title2>
         <.spacing value="M" />
-        <Text.body_large><%= @vm.description %></Text.body_large>
+        <%!-- These fields use the raw() helper to make sure contents from the wysiwyg are rendered --%>
+        <Text.body_large><%= raw(@vm.description) %></Text.body_large>
```

Comment:
These have been tested for XSS using https://gist.github.com/Rokt33r/15665eb75d823a1379eb and no injection succeeded